README.md

```diff
@@ -134,6 +134,12 @@ See [here](https://github.com/Azure/azure-functions-core-tools/blob/master/READM
 
 * [Python 3.6.x](https://www.python.org/downloads/)
 
+### PowerShell
+
```

Comment:
Could we somehow link to your tutorial for debugging in PowerShell? Does that exist yet? Would prefer to have that since attaching breakpoints works differently than other languages

Reply:
Yeah that's a good idea... that should land tomorrow. Can I update that then?

Reply:
Yeah no problem

Reply:
Updated!